CONTRIBUTING.rst

```diff
@@ -0,0 +1,44 @@
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+
+Please read the PAVICS `Developer Documentation`_ to get started.
+
+.. _Developer Documentation: https://birdhouse-deploy.readthedocs.io/en/latest/
+
+Policy
+------
+Since PAVICS is used in production by multiple organizations, this deployment repository also has a policy regarding contributions.
+
+Policy objectives
+~~~~~~~~~~~~~~~~~~~~~
+
+1. Keep production stable between deployments
+2. Encourage PR to be merged more quickly
+3. Achieve the two previous objectives
+      * by weighing down everyone's workflow as little as possible
+      * by having an approach that scales well as more nodes go into production and the number of PR increases
+
+Policy rules
+~~~~~~~~~~~~~~~~~~~~~
+
+1. The repository has a main branch, "master", open to the community where contributions, "PR", are welcome. This master branch must not have owners and therefore no organization can block contributions to it.
+2. Contributions should be backward-compatible whenever possible, or feature a toggle switch so that organizations can activate the new feature on their own schedule.
```

Comment:
I would add a reference to the components and optional-components to give an example of what we mean by toggle switch here.